test/models/erc1155-standard.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { defaultWeb3Connection, getPrivateKeyFromFile } from '../utils/';
+import { ERC1155Standard, Web3Connection } from '../../src';
+import { expect } from 'chai';
+import { Account } from 'web3-core';
+
+describe(`ERC1155 Standard`, () => {
+  let contract: ERC1155Standard;
+  let web3Connection: Web3Connection;
+  let contractAddress: string;
+  let bob: Account;
+  const initial = {
+    uri: 'https://my.token.uri',
+  };
+  before(async () => {
+    web3Connection = await defaultWeb3Connection(true, true);
+    bob = web3Connection.Web3.eth.accounts.privateKeyToAccount(
+      getPrivateKeyFromFile(1)
+    );
+  });
+
+  it(`Deploys the contract`, async () => {
+    const deployer = new ERC1155Standard(web3Connection);
+    deployer.loadAbi();
+    const tx = await deployer.deployJsonAbi('https://my.token.uri');
+    expect(tx.blockNumber).to.exist;
+    contractAddress = tx.contractAddress!;
+  });
+
+  describe(`Methods`, () => {
+    before(async () => {
+      try {
```

Comment:
we can't use try-catch on testing because this would make the _before script_ not fail if we thrown an error and it should

Reply:
Oh yeah you're right